src/script/conversation/ConversationRepository.ts

```diff
@@ -1472,9 +1487,11 @@ export class ConversationRepository {
       }),
     );
 
-    const wasProteus1to1ActiveConversation =
-      !!proteusConversations &&
-      proteusConversations.some(conversation => this.conversationState.isActiveConversation(conversation));
+    // Because of the current architecture and the fact that we present a connection request as a conversation of connect type,
+    // we don't want to inject conversation migrated even if the only proteus 1:1 conversation we had was a connection request.
+    if (!wasProteusConnectionIncomingRequest) {
```

Comment:
We don't want the message to be injected if mls 1:1 was created right after accepting a connection (proteus 1:1 conversation was never used)